pkg/env/env.go

```diff
@@ -441,13 +445,22 @@ func (e *testEnv) execFeature(ctx context.Context, t *testing.T, featName string
 				}
 				ctx = assess.Func()(ctx, t, e.cfg)
 			})
+
+			if e.cfg.FailFast() && t.Failed() {
+				failed = true
+				break
+			}
 		}
 
 		// teardowns run at feature-level
 		teardowns := features.GetStepsByLevel(f.Steps(), types.LevelTeardown)
 		for _, teardown := range teardowns {
 			ctx = teardown.Func()(ctx, t, e.cfg)
 		}
+
+		if e.cfg.FailFast() && failed {
```

Comment:
Should this happen before teardown ?
This issue, #98, seems to want to skip all subsequent steps, including teardown, when fail-fast is signaled.

Reply:
Makes sense. I think this Issue totally skipped my mind when I wrote this. Moving this above the teardown makes sense.